organizations_test.go

```diff
@@ -0,0 +1,84 @@
+package packngo
+
+import "testing"
+
+func TestAccOrg(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c := setup(t)
+	defer organizationTeardown(c)
+
+	rs := testProjectPrefix + randString8()
+	ocr := OrganizationCreateRequest{
+		Name:        rs,
+		Description: "Managed by Terraform.",
+		Website:     "http://example.com",
+		Twitter:     "foo",
+	}
+	p, _, err := c.Organizations.Create(&ocr)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if p.Name != rs {
+		t.Fatalf("Expected new project name to be %s, not %s", rs, p.Name)
+	}
+	rs = testProjectPrefix + randString8()
+	pur := OrganizationUpdateRequest{
+		ID:          p.ID,
+		Name:        rs,
+		Description: "Managed by Terraform.",
+		Website:     "http://quux.example.com",
+		Twitter:     "bar",
+	}
+	p, _, err = c.Organizations.Update(&pur)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if p.Name != rs {
+		t.Fatalf("Expected the name of the updated project to be %s, not %s", rs, p.Name)
+	}
+	gotProject, _, err := c.Organizations.Get(p.ID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if gotProject.Name != rs {
+		t.Fatalf("Expected the name of the GOT project to be %s, not %s", rs, gotProject.Name)
+	}
+	_, err = c.Organizations.Delete(p.ID)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+}
+func TestOrgListPaymentMethods(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	// setup
+	c := setup(t)
+	defer organizationTeardown(c)
+
+	rs := testProjectPrefix + randString8()
+	ocr := OrganizationCreateRequest{
+		Name:        rs,
+		Description: "Managed by Terraform.",
+		Website:     "http://example.com",
+		Twitter:     "foo",
+	}
+	org, _, err := c.Organizations.Create(&ocr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// tests
+	_, _, err = c.Organizations.ListPaymentMethods(org.ID)
```

Comment:
maybe let's do something like 

```go
pms, _, err := c.Organizations.ListPaymentMethods(org.ID)

if err != nil {
    t.Fatal("error: ", err)
}

if len(pms) != 0 {
    t.Fatal("the new test org should have no payment methods")
}
```

.. so that the test does at least something.

It would be really nice if we could create dummy payment method for test under the test org. I tried and it complained about invalid `nonce`. However, I don't see it a blocker for the merge of this PR.